recipe/meta.yaml

```diff
@@ -16,11 +16,14 @@ build:
 
 requirements:
   host:
-    - python >=3.5
+    - python >=3.6
```

Comment:
If we will remove `noarch: python`, then in `host` and `run` should be a plain `python` without pinnings